src/m4/jtr_arm_logic.m4

```diff
@@ -44,11 +44,29 @@ if test "x$enable_native_tests" = xyes; then
     ,[CPU_NOTFOUND="1"]
      [AC_MSG_RESULT(no)]
   )
+  CC="$CC_BACKUP -march=armv8-a+simd"
+  AC_MSG_CHECKING([for ASIMD])
+  AC_RUN_IFELSE(
+    [
+    AC_LANG_SOURCE(
+    [[#include <arm_neon.h>
+  #include <stdio.h>
+  extern void exit(int);
+  int main(){uint32x4_t t;*((long*)&t)=1;t=veorq_u32(t,t);if((*(unsigned*)&t)==88)printf(".");exit(0);}]]
```

Comment:
It's not great that we have C aliasing violations in these configure test programs. I recall us discussing this before (for some other arch), but I don't recall what the outcome of the discussion was.

Reply:
While at that we could drop the need for stdio and just end with a calculated exitcode, expected 0 - the current test of "88" is most everywhere and doesn't test correctness, we only look for build ok/fail and run ok/segfault. Let's make a separate issue for it.